src/main/java/mate/academy/repository/MySqlBookDao.java

```diff
@@ -0,0 +1,120 @@
+package mate.academy.repository;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class MySqlBookDao implements BookDao {
+    private static final int ID_PARAMETER_INDEX = 1;
+    private static final int TITLE_PARAMETER_INDEX = 1;
+    private static final int PRICE_PARAMETER_INDEX = 2;
+    private static final int UPDATE_ID_PARAMETER_INDEX = 3;
+    private static final int ZERO_AFFECTED_ROW = 0;
+    private static final int ONE_AFFECTED_ROW = 1;
+    private static final String ID_ROW_LABEL = "id";
+    private static final String TITLE_ROW_LABEL = "title";
+    private static final String PRICE_ROW_LABEL = "price";
+
+    @Override
+    public Book create(Book book) {
+        String query = "INSERT INTO book(title, price) VALUES(?, ?)";
+        try (Connection connection = MySqlConnectionManager.getConnection();
+             PreparedStatement statement = connection.prepareStatement(query,
+                     Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(TITLE_PARAMETER_INDEX, book.getTitle());
+            statement.setBigDecimal(PRICE_PARAMETER_INDEX, book.getPrice());
+            if (statement.executeUpdate() < ONE_AFFECTED_ROW) {
```

Comment:
Actually, it's a very rare case when you can receive 0 affected rows on insert, not sure if this check is very useful.